Formula/b/bsc.rb

```diff
@@ -27,12 +40,12 @@ class Bsc < Formula
   depends_on "pkgconf" => :build
   depends_on "gmp"
   depends_on "icarus-verilog"
-  depends_on "tcl-tk@8"
 
   uses_from_macos "bison" => :build
   uses_from_macos "flex" => :build
   uses_from_macos "libffi"
   uses_from_macos "perl"
+  uses_from_macos "tcl-tk"
```

Comment:
macOS `tcl-tk` is old and deprecated; we probably shouldn't be using it anymore.

Reply:
Would need upstream changes as they specifically set to system copy - https://github.com/B-Lang-org/bsc/blob/main/platform.sh#L81-L86
```sh
if [ ${OSTYPE} = "Darwin" ] ; then
    # Have Makefile avoid Homebrew's install of tcl on Mac
    TCLSH=/usr/bin/tclsh
else
    TCLSH=`which tclsh`
fi
```

Reply:
Probably worth clarifying with upstream if this change, [introduced over 4 years ago](https://github.com/B-Lang-org/bsc/pull/427), still makes sense, especially since:-
```
$ /usr/bin/tclsh

WARNING: This version of tcl is included in macOS for compatibility with legacy software.
In future versions of macOS the tcl runtime will not be available by
default, and may require you to install an additional package.

% 
```

Reply:
Upstream should be aware that system Tcl is outdated and has mentioned interest in moving away from it back in 2022:
- https://github.com/B-Lang-org/bsc/issues/504#issuecomment-1286287406

Issue author just patched out the system Tcl logic like: https://github.com/kammoh/bsc/commit/ded6d778ca348442d3c6fec577fb0ae890b3e08c